tests/init.c

```diff
@@ -141,6 +141,14 @@ int main (int argc, char **argv)
       exit (0);
     }
 
+  if (strcmp (argv[1], "printenv") == 0)
```

Comment:
Don't know if it's important or not, but printenv exit code:

```
 0 if all variables specified were found
 1 if at least one specified variable was not found
 2 if a write error occurred
```

It may not matter for your tests.